sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusProcessorOptions.cs

```diff
@@ -141,6 +141,11 @@ public int MaxConcurrentCalls
         /// </summary>
         public string Identifier { get; set; }
 
+        /// <summary>
+        /// Number of messages to receive.
+        /// </summary>
+        public int BatchSize { get; set; } = 1; // Should be an integer > 1 and less than some limit.
```

Comment:
are we checking that this value is greater than 0?

Reply:
updated!